### PR TITLE
add new app: de.halaszovich.PMbrowser

### PR DESCRIPTION
**Pull request by ChrisHal**
I can confirm all criteria are met

### Please confirm your submission meets all the criteria

- [ +] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [ +] My pull request follows the instructions at [App Submission][submission].
- [ +] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [ +] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ +] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ +] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ +] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: de.halaszovich.PMbrowser.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "app-id": "de.halaszovich.PMbrowser",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.3",
+  "sdk": "org.kde.Sdk",
+  "command": "QtPMbrowser",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
```
Comment (hfiguiere) [inline de.halaszovich.PMbrowser.json:10]:
```suggestion
    "--socket=fallback-x11",
    "--socket=wayland",
```

File: de.halaszovich.PMbrowser.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "--filesystem=host",
```
Comment (hfiguiere) [inline de.halaszovich.PMbrowser.json:11]: this is a very broad filesystem permission.

**Comment by hfiguiere** [comment]
bot, build de.halaszovich.PMbrowser

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: de.halaszovich.PMbrowser.json (hunk 2 above)
Comment (ChrisHal) [reply de.halaszovich.PMbrowser.json:11]: Okay, I will try to restrict the permission as much as possible. The user will want to load files and export data, not all of which can be can be handled by the portal mechanism.

**Comment by ChrisHal** [comment]
bot, build

**Comment by hfiguiere** [comment]
You need to specify the app id. Also it would be nice to have a clear title in the PR. 

bot, build de.halaszovich.PMbrowser

**Comment by ChrisHal** [comment]
I tested the build and everything works as expected. 

Thanks for the time spent reviewing this!

**Comment by ChrisHal** [comment]
Please let me know if there is anything left I would need to do.

**Pull request closed without merging**